APIBacklog/documentation/SupportingDocuments/API proposals/APIproposal_BlockchainPublicAddress_Telefonica.md

```diff
@@ -0,0 +1,12 @@
+| **Field** | Description | 
+| ---- | ----- |
+| API family name | Blockchain Public Address |
+| API family owner| Telefonica  |
+| API summary | **High level description**<br> This API allows to manage a Blockchain Public Address associated to a phone number, i.e. to retrieve the blockchain public address(es) and to bind/unbind a Blockchain Public Address. The Block Public Address is referred to as Decentralized IDentifier (DID). With the proposed API, telco service providers have the opportunity to provide 3rd parties with the following capability: pairing phone number with DID whenever this 3rd party wants to offer its customers a way to make transactions more easily based on the phone number instead of the DID. <br><br>**Rationale**<br>DID is composed of 26-35 alphanumeric characters in length. This identifier is not memorisable or usable by a person, so there is the possibility of positioning the telephone number as the predominant identifier on web3. This new identity protocol will make it possible to have a system similar to the Telcos' bizum in web3 ecosystem.
```

Comment:
“The Block Public Address is referred to as Decentralized IDentifier (DID)” is not quite right, as there are DID methods that not even use blockchain (e.g. did:web and did:dns). Maybe rephrase as 'This Block public address is utilized as  Decentralized Identifier'

Reply:
Text is updated according to your suggestion in this commit: https://github.com/camaraproject/WorkingGroups/pull/154/commits/6bec4f9f8a0c0a35295992f9c57c2c4ced689ad5